charts/unpoller/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v2
+name: unpoller
+description: A Helm chart for unpoller, a unifi prometheus exporter
+type: application
+version: "2.11.2"
+appVersion: "v2.11.2"
+keywords:
+  - esp32
+  - home-assistant
```

Comment:
What are these?

Reply:
I deny having copy pasted the structure from other chart and forgetting fixing it. :rofl: 
fixed.